src/ssl.c

```diff
@@ -1960,6 +1960,20 @@ int wolfSSL_SNI_GetFromBuffer(const byte* clientHello, word32 helloSz,
 #endif /* HAVE_SNI */
 
 
+#ifdef HAVE_TRUSTED_CA
+
+WOLFSSL_API int wolfSSL_UseTrustedCA(WOLFSSL* ssl, byte type,
+            const byte* cert, word32 certSz)
+{
+    if (ssl == NULL)
```

Comment:
Is a NULL cert really valid?

Reply:
It looks like only the size was eventually checked, not the cert pointer. I added more complete parameter checking.